docs/index.storybook.mdx

```diff
@@ -1,60 +1,298 @@
+    }
+
+    .main-content {
+      position: relative;
+      z-index: 10 !important;
+    }
+    
+    .heading-red {
+      color: #CA0928;
+      font-size: 32px;
+    }
+
+    .search-indicator {
+      color: #C6CDD4;
+      font-weight: 600;
+      margin-left: 24px;
+    }
+
+    .key-indicator {
+      color: white;
+      border-radius: 8px;
+      background: #C6CDD4;
+      padding: 4px 10px;
+      margin: 0 4px;
+    }
+
+    .hero-banner {
+      position: relative;
+      overflow: hidden;
+    }
+
+    .hero-illustration {
+      height: 300px;
+      position: absolute;
+      right: 0;
+      top: 50%;
+      transform: translate(0, -50%);
+    }
+
+    .card {
+      width: 300px;
+      min-width: 300px;
+    }
+
+    .card-half-width {
+      width: 475px;
+      min-width: 475px;
+    }
 
-<Meta title="Get Started" />
+    .icon-wrapper {
+      min-width: 40px;
+      width: 40px;
+      min-height: 40px;
+      height: 40px;
+      border-radius: 24px;
+      background: #CA0928;
+    }
 
-# Wave Design System
+    .icon-wrapper-white {
+      background: #FFFFFF;
+    }
+  `}
+</style>
 
-Welcome to the Design System of FREE NOW. Thanks for stopping by! This is the place where you find the latest components and essentials, that enable you to rapidly build your product.
+<img src={WaveBackground} className="wave-bg" width="100%" alt="wave-background" />
 
-## Installation
+<Box className='main-content'>
+  {/* Logo */}
+  <Box className='sb-unstyled' display='flex' mb='40px'>
+    <img src={WaveLogo} width='170px' alt='wave-logo' />
+    <img src={ByFreenow} width='170px' alt='by-freenow-logo' />
+  </Box>
 
-To bring the design system into your project, install it as a package:
+  <Card className='hero-banner' level={200}>
+    <Box display='flex' flexDirection='column'>
+      <Headline style={{ lineHeight: '40px', marginBottom: '16px' }}><span className='heading-red'>Develop experiences faster</span> <br/>with our UI component library</Headline>
 
-```bash
-npm i @freenow/wave
-```
+      <Text>Our Wave design system offers a comprehensive suite <br/>of UI components to help you ship new features faster.</Text>
 
-## Usage
+      <Box className='sb-unstyled' display='flex' mt='24px' alignItems='center'>
+        <a href='/?path=/docs/get-started--docs'>
+          <Button>Get started &nbsp; <ArrowRightIcon /></Button>
+        </a>
 
-All of our components are exported by name from `@freenow/wave`, so you can import them with:
+        <div className='search-indicator sb-unstyled'>
+          Or press <span className='key-indicator'>/</span> to search components
+        </div>
+      </Box>
+    </Box>
 
-```jsx
-import { Card, Headline, Text } from '@freenow/wave';
+    <img src={Map} className='hero-illustration' alt='map-illustration' />
 
-export default () => (
-    <Card level={200}>
-        <Headline>Welcome</Headline>
-        <Text>FREE NOW Design System</Text>
+  </Card>
+
+{' '}
```

Comment:
I see this repeated multiple times and I'm not sure what it's purpose is 😕

Reply:
I think I had this auto-generated, will remove 😅